Added receptorctl work release documentation (#1061)
ansible/receptor: docs/source/receptorctl/receptorctl_work_release.rst

------------
work release
------------

.. contents::
   :local:

> ``receptorctl work release`` Deletes one or more units of work.

Reviewer: ```suggestion
``receptorctl work release`` deletes one or more units of work.
```

Command syntax: ``receptorctl --socket=<socket_path> work release [<<Options>>] <<Unit ID>> [...]``

``socket_path`` is the control socket address for the Receptor connection.
   The default is ``unix:`` for a Unix socket.
   Use ``tcp://`` for a TCP socket.
   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.

.. code-block:: text

  ss --listening --processes --unix 'src = unix:<socket_path>'
  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))

``ps -fp $(pidof receptor)``
``lsof -p <pid>``

> ``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work release`` command, you should specify the ``Unit ID`` for the Receptor instance to which you are connected.

Reviewer: ```suggestion
``Unit ID`` is a unique identifier for a work unit (job).  When running the ``work release`` command, you should specify the ``Unit ID`` for the Receptor node to which you are connected.
```

Reviewer: Hmm. Just noticed this but it might be better to `s/instance/node` to be more consistent with the terminology throughout the docs. I guess this also needs to be changed here too: https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_work_cancel.rst

> Options:

Reviewer: ```suggestion
```

Reviewer: The word "Options:" seems like it is dangling here. I also don't see it in other docs like this one: https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_ping.rst

If you'd like to add that, I'd recommend dropping the colon `:` and turning it into a subheading or possibly a table.

> ``--all`` will delete all work units.

Reviewer: ```suggestion
``--all`` deletes all work units.
```

> ``--force`` will delete locally (even if the remote node is unreachable).

Reviewer: ```suggestion
``--force`` deletes work units locally and takes effect even if the Receptor node is unreachable.
```

Author: Proposed alternate changes in next commit
